Makefile

```diff
@@ -185,7 +185,7 @@ COCKPIT_REPO_FILES = \
 	$(NULL)
 
 COCKPIT_REPO_URL = https://github.com/cockpit-project/cockpit.git
-COCKPIT_REPO_COMMIT = 47539d6d31520da8b0ed916ee6d78e5aefc1ea87
+COCKPIT_REPO_COMMIT = 8f48c4740593e4eb08e1f40ffd98dbeb09f5c14a
```

Comment:
I'd really prefer this was a tag name, or at least get an accompanying comment that says "# 270". SHAs are not for humans.

Reply:
https://github.com/cockpit-project/cockpit-podman/commit/cfc05e730d6f5ddf4244d111bf2d7d0b0c24cc4b

as per commit message:
> ...
> We'll hardcode shas from here on out: it's more secure (ie:
tamperproof), and more performant (since we don't have to check the ref
if we already have the object), in addition to being easier to work with
in terms of the required git commands.

Reply:
Sure, but that doesn't prevent us from adding a comment.